docs/conf.py

```diff
@@ -54,18 +54,19 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'pysatSpaceWeather'
-copyright = u'2020, Angeline G Burrell, Jeffrey Klenzing, Russell Stoneback'
-author = u'Angeline G Burrell, Jeffrey Klenzing, Russell Stoneback, Asher Pembroke, Carey Spance'
+project = 'pysatSpaceWeather'
+proj_desc = '{:s} Documentation'.format(project)
+zenodo = json.loads(open('../.zenodo.json').read())
+author = ', '.join([x['name'] for x in zenodo['creators']])
+manual_copyright = ', '.join(['2021', author])
```

Comment:
Style question: Sphinx expects a "copyright" keyword in this file.  If we use "manual_copyright" here, the copyright line at the bottom of the html will be blank.  Just discovered this while working out documentation bugs in the core code.  Do we have a project preference here?

Reply:
I changed from `copyright` since it overwrites an existing variable.  But if this is expected by sphinx, then let's change it back.